implement order queue page

ibdthor opened the pull request:
order queue page and its accompanying css file related to issues #49
and #50

ibdthor committed:
implement order queue page

order queue page and its accompanying css file related to issues #49
and #50

cmontella commented:
Just before I look at it, rename the file back to `*.eve.md` so that Eric's changes will merge in

ibdthor committed:
improved order queue and appended .md

changed file type back to .md and added some extra literate portions to
the order queue code

ibdknox committed:
Merge branch 'master' into order-queue

joshuafcole committed:
Merge with master

joshuafcole commented:
🍶  🇳🇪  🌃
